walletconnect/main.js

```diff
@@ -0,0 +1,179 @@
+import {
+  AccountId,
+  TokenAssociateTransaction,
+  TokenId,
+} from "@hashgraph/sdk";
+import {
+  DAppConnector,
+  HederaJsonRpcMethod,
+  HederaSessionEvent,
+  HederaChainId,
+} from "@hashgraph/hedera-wallet-connect";
+
+// Global state for wallet connection
+let accountId = '';
+let isConnected = false;
+
+// Network configuration
+const NETWORK_CONFIG = {
+  testnet: {
+    network: "testnet",
+    jsonRpcUrl: "https://testnet.hedera.validationcloud.io/v1/GYSdi5Rlhc-NmoBLSVJGsqVQDOL6C4lCCQbyHc3NvsM",
```

Comment:
I assume that the string after `/v1/` is an API key, if so, this key has leaked - not good practice to do so.

I suggest that you read the RPC URL from a `.env` file, and if not present default to hashio (which does not use API keys).

Reply:
@bguiz  That was may bad. I have updated the template to use hashio.